### ISSUE TITLE
508-defect-2 [FOCUS]: Upload and delete file buttons should have circular focus

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24504]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a keyboard and screen reader user, I expect focus to be managed when interacting with the upload and delete file buttons.

## Details

Focus is lost after the user uploads and deletes a file. For screen reader users, that means being taken out of their current context. For keyboard users, that means losing sight of the visible focus halo.

https://user-images.githubusercontent.com/14154792/117758608-5b7b3280-b1f0-11eb-8314-ffa27b3939c3.mov

## Acceptance Criteria

- [ ] Upon using the upload button, focus is sent to the document name field and is visible
- [ ] Upon deleting a file, focus is circular and returns to the "upload" button and is visible

**Comment by joshkimux** [comment]
These are the same! Please pick the one you'd prefer to close out 😄 
Thank you for bringing this to my attention ❤️

**Comment by hdjustice** [comment]
Closing this ticket out as a duplicate of #27410, We will address this issue there.

**Comment by hdjustice** [comment]
Thank you for taking a peek at this one @joshkimux .  We closed it out and will address in #27410.
